packages/synapse-react-client/src/style/components/_global-alert.scss

```
@@ -1,6 +1,6 @@
 .SynapseNavDrawerIsShowing .FullWidthAlertGlobal {
-  width: 100%;
-  .FullWidthAlert[role='alert'] {
-    margin-left: 56px;
-  }
+  width: calc(100vw - 56px);
+  left: 56px;
```

Using an scss variable would clarify why we have this style
```suggestion
  $-synapse-nav-drawer-width: 56px;
  width: calc(100vw - $-synapse-nav-drawer-width);
  left: $-synapse-nav-drawer-width;
```